app/__init__.py

```diff
@@ -41,7 +41,7 @@
                   output='gen/main_packed.%(version)s.css')
 assets.register('main_css', main_css)
 
-main_js = Bundle('js/plugins.js',
+main_js = Bundle('js/main.js',
```

Comment:
cool, I didn't even know about [Flask-Assets](https://flask-assets.readthedocs.org/en/latest/), or that we were using it!